src/components/play/PlayHints.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+
+import { Alert, AlertTitle } from '@mui/material';
+
+import { HINTS_PLAY_CY } from '../../config/selectors';
+import { QUIZ_TRANSLATIONS } from '../../langs/constants';
+
+type Props = {
+  hints: string | undefined;
+  maxAttempts: number;
+  currentAttempts: number;
+  isCorrect: boolean;
+  mt?: number;
+};
+
+export const PlayHints = ({
+  hints,
+  maxAttempts,
+  currentAttempts,
+  isCorrect,
+  mt,
+}: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    hints &&
+    !isCorrect &&
+    currentAttempts > 0 &&
+    currentAttempts < maxAttempts && (
```

Comment:
Maybe it would be better to use an `if` to make this explicit ? and `return null` otherwise.